### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by hienphan0111**
This section, I have completed the following requirements:
- [ ] Add any missing JSX elements to your project (without adding extra functionality)

- [ ] Final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/components/Navbar.component.jsx [hunk 1]
```diff
@@ -5,10 +5,17 @@ function Navbar() {
...
+          <h1>Bookstore CMS</h1>
+          <div className="links">
+            <Link className="link" to="/">Book</Link>
+            <Link className="link" to="categories">Categories</Link>
+          </div>
+        </div>
+        <div className="user">
+          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 i-user">
+            <path strokeLinecap="round" strokeLinejoin="round" d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z" />
+          </svg>
```
Comment (roshan-bajgain) [inline src/components/Navbar.component.jsx:18]: - [ ] Styling should be exactly the same provide on the [Zeplin](https://zeplin.io/). Please make sure to add the same color, font, and spacing. Please see the image below for reference.
- [ ] I would suggest you please use the credentials provided on your requirement 
### Your design
his

### Original design
original

File: src/components/Book.component.jsx [hunk 2]
```diff
@@ -13,12 +14,30 @@ function Book({ book, id }) {
...
+        <div>
+          <h4>64%</h4>
+          <span>Completed</span>
+        </div>
+      </div>
+      <div className="sec-3">
+        <span>Current chapter</span>
+        <h4>Chapter 17</h4>
+        <button type="button">update progress</button>
       </div>
```
Comment (roshan-bajgain) [inline src/components/Book.component.jsx]: - [ ] According to the project requirement your styling should be exactly the same as provided on the Zeplin. Please make sure to add the same color and font. Please see the image below for reference.
- [ ] Please use the same color and font provided on the design
### Your design
his

### Original design
original

File: src/components/Book.styles.scss [hunk 3]
```diff
@@ -2,24 +2,92 @@
...
+      .category {
+        font-weight: 400;
+        color: #12121234;
+      }
+
+      .author {
+        color: #4386bf;
+      }
+    }
+
```
Comment (roshan-bajgain) [inline src/components/Book.styles.scss]: - [ ] I would suggest you please make the input field matches the design. like color, font and the input is covering the whole width. Please design below for reference. Please make the color same for`Add new Book, Placeholder` 
### Your design
his

### Original design
original

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @hienphan0111, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED 👍 

Hi @hienphan0111 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Styling exactly as the Zeplin design 
:heavy_check_mark: CSS best practices used
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**